src/Microsoft.DotNet.Configurer/IAspNetCertificateSentinel.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.Configurer
+{
+    public interface IAspNetCertificateSentinel : IDisposable
```

Comment:
You don't need for it to implement IDisposable right?

Reply:
IFirstTimeUseNoticeSentinel and FirstTimeUseNoticeSentinel declared, so I just followed along, but neither this nor firstimeusenoticesentinel need them. I'll remove it from this one and I can remove it from the other one if you want me too.